spec/alephant/logger/level_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper'
+require 'alephant/logger/level'
+require_relative 'support/level_shared_examples'
+
+RSpec.describe Alephant::Logger::Level do
+  subject { described_class.new(log_level) }
+
+  describe '#logs?' do
+    context 'Message level' do
```

Comment:
Looking at these level specs, maybe my understanding is wrong. When the logger is set to warn, shouldn't it then log all messages that are warnings or errors and not debug and info? If this is the case, warn and error should behave like the writable log and the info and debug should be the non-writable log?

Reply:
@samfrench The current context refers to the `Message level` i.e. the user defined level. The `log_level` refers to the defined methods i.e. `debug, info, warn, error`. Thus for this context, we're testing against the defined method `:info` while altering the scale which is the message level.

Since we're testing against dynamic methods; this is a necessary check. E.g.
```ruby
def info; end # log_level 
message_level # when :debug ; should allow info to write
```

Reply:
When message level is set to `warn`. it should not write for the log level `:info` since that's lower. i.e `writeable?` should return false for `:info`. https://github.com/BBC-News/alephant-logger-json/pull/7/files/e9079bb3866d951c2407257569612bfa99c63e4c#diff-70801f1f5d1a4cfecf35893f2fa2b3b9R28

Reply:
I think I can improve the language to make it clearer.